tests/deprecated_api/test_remove_1-3.py

```diff
@@ -18,38 +18,10 @@
 import pytest
 import torch
 
-from pytorch_lightning import LightningModule, Trainer
-from pytorch_lightning.callbacks import EarlyStopping, ModelCheckpoint
+from pytorch_lightning import Trainer
 from pytorch_lightning.profiler.profilers import PassThroughProfiler, SimpleProfiler
 
 
-def test_v1_3_0_deprecated_arguments(tmpdir):
-    with pytest.deprecated_call(match='will no longer be supported in v1.3'):
-        callback = ModelCheckpoint()
-        Trainer(checkpoint_callback=callback, callbacks=[], default_root_dir=tmpdir)
-
-    # Deprecate prefix
-    with pytest.deprecated_call(match='will be removed in v1.3'):
-        ModelCheckpoint(prefix='temp')
-
-    # Deprecate auto mode
-    with pytest.deprecated_call(match='will be removed in v1.3'):
-        ModelCheckpoint(mode='auto')
-
-    with pytest.deprecated_call(match='will be removed in v1.3'):
-        EarlyStopping(mode='auto')
-
-    with pytest.deprecated_call(match="The setter for self.hparams in LightningModule is deprecated"):
-
-        class DeprecatedHparamsModel(LightningModule):
-
-            def __init__(self, hparams):
-                super().__init__()
-                self.hparams = hparams
-
-        DeprecatedHparamsModel({})
```

Comment:
if you remove the test you probably also want to remove the warning message where it is produced, i.e., removing the setter entirely.

Reply:
Done in 2b6bdaf

Reply:
yes, maybe I just missed it :]

Reply:
oh, not this one yet, good catch